flows/actions/testdata/send_msg.json

```diff
@@ -419,21 +419,17 @@
             "text": "Hi Ryan Lewis, who's a good boy?",
             "templating": {
                 "uuid": "9c4bf5b5-3aa4-48ec-9bb9-424a9cbc6785",
+                "template": {
+                    "uuid": "5722e1fd-fe32-4e74-ac78-3cf41a6adb7e",
+                    "name": "affirmation"
+                },
                 "params": {
                     "body": [
                         "@contact.name",
                         "boy"
                     ],
                     "uuid": "1067f8e2-82f0-4378-9214-0f019365ddb7"
-                },
-                "template": {
-                    "uuid": "5722e1fd-fe32-4e74-ac78-3cf41a6adb7e",
-                    "name": "affirmation"
-                },
-                "variables": [
-                    "@contact.name",
-                    "boy"
```

Comment:
bit weird having tests where there is both `params` and `variables`.. that shouldn't happen

Reply:
I thought we support both until the editor is updated, alternatively we need to have the flow migrations to make sure only use params

Reply:
Yes the editor should support flows with either, but there will never be a single flow that contains both.